upup/pkg/fi/cloudup/populate_cluster_spec.go

```diff
@@ -297,6 +297,7 @@ func (c *populateClusterSpec) run() error {
 			codeModels = append(codeModels, &components.KubeDnsOptionsBuilder{Context: optionsContext})
 			codeModels = append(codeModels, &components.KubeletOptionsBuilder{Context: optionsContext})
 			codeModels = append(codeModels, &components.KubeControllerManagerOptionsBuilder{Context: optionsContext})
+			codeModels = append(codeModels, &components.CloudControllerManagerOptionsBuilder{Context: optionsContext})
```

Comment:
Can you comment this out, so we are not wiring it in?  Let's add it in when nodeup supports it.

Reply:
There is a sentinel inside the builder to skip if ExternalCloudController is not enabled. Would that suffice?